app/models/flood_risk_engine/comment.rb

```diff
@@ -0,0 +1,6 @@
+module FloodRiskEngine
+  class Comment < ActiveRecord::Base
+    belongs_to :commentable, polymorphic: true
+    belongs_to(:user) if defined?(User)
```

Comment:
If you move `Comment` to BO you have problems with the joins from objects using the comments, such as `EnrollmentExemption` (that's where I started). 

It looks to me like we should move all models to the engine - because of the issues around database joins being defined across the engine and host app.